src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+  
+  <title>Document</title>
+</head>
+<body class="d-flex flex-column align-items-center">
+  <main class="w-25 mx-auto my-auto">
+    <div id="todo-header" class="d-flex justify-content-between align-items-center m-auto p-2">
+      <h1 class="fs-3 text-secondary">All Tasks</h1>
+      <img src="#" class="icon sync-icon fas fa-sync-alt text-secondary" alt=""/>
+    </div>
+    <form class="m-auto px-2 d-flex position-relative">
+      <input id="add-task" class="w-100 p-2" type="text" placeholder="Add a task and press Enter key">
+        <div class="arrow-icon position-absolute"><i class="fas fa-2x fa-arrow-right"></i></div>
+      <img src="#" id="arrow-icon" class="icon arrow-icon position-absolute fas fa-2x fa-arrow-right" alt=""/>
+    </form>
+    <div id="todo-list-container" class="m-auto my-2 px-2">
+    </div>
+    <div>
+    <button id="clear-tasks" type="button" class="btn btn-light w-100 text-secondary">Clear all Tasks</button>
+    </div>
```

Comment:
I suggest you use the `section` tag instead `div`, which will be more specific in showing the skeletal system of the page🤗

Reply:
Thank you, here I  have only one section, that is why used div.

Reply:
Alright, but if you inspect the page with your browser, you will see the skeletal system am talking about, I still suggest you use `section` in place of `div`.